pontoon/base/views.py

```diff
@@ -979,9 +979,11 @@ def user_data(request):
 
     return JsonResponse({
         'is_authenticated': True,
+        'is_admin': user.has_perm('base.can_manage_project'),
```

Comment:
Shouldn't that be true only if the user is a superuser? There are non-superusers who can access the admin interface?

Reply:
Yes. Superusers can also access Django Admin, which gives users more power, among other things to delete stuff. We have external PMs (for projects like MDN), which can only access Pontoon Admin.

Still, this is confusing, and hence we have bug 1203788 & 1204659.